data/mods.json

```diff
@@ -494,5 +494,1461 @@
+  "Firebrand",
+  "Fris0uman",
+  "Hirmuolio",
+  "Honcharuk",
+  "HuXTUS",
+  "I-am-Erk",
+  "Icecoon",
+  "Inglonias",
+  "Jackledead",
+  "Jebi Bones",
+  "Jninja1",
+  "KS",
+  "KantoJolto",
+  "Kettle Swordfang",
+  "Kevin Giovinazzo",
+  "Kevin",
+  "Kilvoctu",
+  "Kodi Arfer",
+  "KorGgenT",
+  "LaVeyanFiend",
+  "KorGgenT?",
+  "Kurchie",
+  "Lanceo90",
+  "Larwick",
+  "Leon",
+  "Maddremor",
+  "Marte",
+  "Mary",
+  "Medecube",
+  "Mewb & Maddremor",
+  "Mewb",
+  "Mikazu",
+  "Minecrafter  Next Generation",
+  "Miso stewed eggplant",
+  "MkMaxxi",
+  "Mokou-tan",
+  "MrAbrahms",
+  "MrJet",
+  "Mshock777",
+  "Xotto",
+  "MutaMan",
+  "Nerdnub",
+  "Night_Pryanik",
+  "Noctifer",
+  "Noctifer-de-Mortem",
+  "Nphyx",
+  "Oa",
+  "Nonesuch",
+  "Olaroll",
+  "Photoloss",
+  "Psychodata",
+  "Caring-Goat-Deity",
+  "Poragon",
+  "ProfoundDarkness",
+  "Richard & Malkeus",
+  "RickyVanz",
+  "Rivet-the-Zombie",
+  "Rod99",
+  "Rod995",
+  "RogueYun",
+  "Rooki1",
+  "community",
+  "Ryan \"DeNarr\" Saige",
+  "Kevin Granade",
+  "STK2008",
+  "ShadowDragon8685",
+  "Slava 'zveroboy' Malikov",
+  "Soupster89",
+  "Spencer Michaels",
+  "Stopsignal",
+  "Stormtrooper",
+  "TGWeaver",
+  "TK90",
+  "TechyBen",
+  "Tefnut",
+  "Shard",
+  "The Goat God",
+  "The Sly Somnambulist",
+  "TheMurderUnicorn",
+  "Theundyingcode",
+  "daftfad",
+  "TinyHippo",
+  "Tokiko1",
+  "Toltetix",
+  "Tonkatsu",
+  "Trivega & Community",
+  "Trivega",
+  "TwinCrows",
+  "Undereastern",
+  "Uvi",
+  "Vermilion",
+  "Vinicius",
+  "WHALEY",
+  "Wayticus",
+  "Williham Totland",
+  "Wulfle",
+  "Xarit",
+  "Xhuis",
+  "YasuYasu",
+  "Yonsisac",
+  "YukariMindGap",
+  "Zaweri",
+  "Zetsukaze",
+  "acidia",
+  "anothersimulacrum",
+  "begginfokillz",
+  "crom",
+  "BorkBorkGoesTheCode",
+  "dambuk1",
+  "darktoes",
+  "dvolk",
+  "e1ven",
+  "ephemeral_storyteller",
+  "Kenan",
+  "esotericist",
+  "AnthonBerg",
+  "harison86",
+  "hount of '@'",
+  "inoue&Derp",
+  "mugling",
+  "no name",
+  "noahgoldfox",
+  "nopeparty",
+  "nornagon",
+  "redxlaser15",
+  "robotic_gamer",
+  "sanyaskillpro",
+  "tungtn",
+  "tungtn/community",
+  "vegarbg",
+  "yy376",
+  "味噌煮込み茄子",
+  "AseaHeru",
+  "Malkeus",
+  "Ornolf",
+  "Artyom177",
+  "Leon713",
+  "and /u/chieflean",
+  "Broken \"Spaghetti\" Admin",
+  "Everyone",
+  "Hio",
+  "Kane",
+  "LambdaSix",
+  "Sunshine",
+  "TheSalamanderDJ",
+  "anon",
+  "xXenocage",
+  "MATTYMCAIN",
+  "Community",
+  "Eirenex",
+  "Fire man - Translated by Chesthole",
+  "GiMoe",
+  "GovTD/Karl_Keimov",
+  "Chaosvolt",
+  "Aptronym",
+  "Mrsecretguy1/Kenan",
+  "SomeDeadGuy",
+  "Axema Vales",
+  "Peeved Penguin (Original Author)",
+  "Bl4nk (Fixer)",
+  "Snofielf (Fixer)",
+  "Whaley",
+  "pisskop",
+  "Dissociativity",
+  "Goat-God",
+  "pixel and Eirenex",
+  "SG",
+  "Fuji",
+  "Unknown",
+  "pisskop and Noctifer"
+],
+    "size": 28034068,
+    "url": "https://github.com/TheGoatGod/Community-Mod-Compilation/archive/v0.1.5.zip",

```

Comment:
"url": "https://github.com/GMC-Modding-Team/Community-Mod-Compilation-redux/releases/tag/v0.1.6"
this will change a lot over the next few months tbh i can make a pr every time I release a new version (if i remember)

Reply:
changes to latest release